runtime/src/accounts_db.rs

```diff
@@ -196,6 +332,8 @@ pub struct AccountStorageEntry {
     /// This is used as a rough estimate for slot shrinking. As such a relaxed
     /// use case, this value ARE NOT strictly synchronized with count_and_status!
     approx_store_count: AtomicUsize,
+
+    alive_bytes: AtomicUsize,
```

Comment:
nits: hopefully I would love some minimum test coverage for this new field..

Reply:
test `test_alive_bytes` added!